src/java.base/linux/classes/sun/nio/fs/LinuxFileSystem.java

```diff
@@ -121,10 +127,56 @@ List<UnixMountEntry> getMountEntries() {
         return getMountEntries("/etc/mtab");
     }
 
-
-
     @Override
     FileStore getFileStore(UnixMountEntry entry) throws IOException {
         return new LinuxFileStore(this, entry);
     }
+
+    // --- file copying ---
+
+    @Override
+    protected void bufferedCopy(int dst, int src, long address,
```

Comment:
Everything is in sun.nio.fs and the existing methods aren't protected. I don't mind if we have the protected methods but it's a bit inconsistent to have it on some but not all of the methods that are intended to be used by subclasses.

Reply:
I was thinking that `protected` would suggest that the method should likely be overridden.

Reply:
Removed `protected` in b6cb94901d1bbaa34917a5aaf381ecf700282c54.